### ISSUE TITLE
Show the length of effect

### ISSUE THREAD

**Issue by GuardianDll** [CleverRaven/Cataclysm-DDA#55185]
### Is your feature request related to a problem? Please describe.

Some effects (mostly magiclysm one, but who knows) has very known length of effect, but it would be really cool if you can do it right on character menu

### Solution you would like.

New flag or field for effects to show their length like in dev mode. Turned off by default, of course

**Comment by NetSysFire** [comment]
I agree that it would be nice and it can also probably be used for other things, too, like the flu or the cold. "It seems to be getting better", "It is almost gone", with customizable messages and a flag if the progress should be exposed or not.

**Comment by LeahLuong** [comment]
kevin closed #54337 after stating:

> Actually our policy is not to accept debug feature requests, so closing.

**Comment by NetSysFire** [comment]
You misread the issue.

>New flag or field for effects to show their length **like in dev mode.**
